pkg/controller/compliance/compliance_controller_test.go

```diff
@@ -173,11 +178,14 @@ var _ = Describe("Compliance controller tests", func() {
 
 		By("replacing the cert with one that has the wrong DNS names")
 		Expect(c.Delete(ctx, &corev1.Secret{ObjectMeta: metav1.ObjectMeta{Name: render.ComplianceServerCertSecret,
-			Namespace: render.OperatorNamespace()}})).NotTo(HaveOccurred())
+			Namespace: rmeta.OperatorNamespace()}})).NotTo(HaveOccurred())
 
 		oldDNSName := "compliance.tigera-compliance.svc"
-		newSecret, err := render.CreateOperatorTLSSecret(nil,
-			render.ComplianceServerCertSecret, render.ComplianceServerKeyName, render.ComplianceServerCertName, render.DefaultCertificateDuration, nil, oldDNSName,
+		ca, err := tls.MakeCA(rmeta.DefaultOperatorCASignerName())
```

Comment:
Because I don't think we should be generating tls certificates in the render package, I'd like to separate these concerns. We can achieve the same thing by having a convenience function in the controller that creates the certificate and passes it to the render function, but I just chose to do this instead since it since it was just an extra line. I can make a convenience function if you'd like in the controllers package.

Reply:
I'm talking about why do MakeCA when CreateTLSSecret will do the call for us if we pass nil CreateTLSSecret.
I don't understand how directly doing MakeCA here is a step in the direction you're suggesting of moving cert creation out of render.

Reply:
I've reverted this change as per our conversation on zoom and we'll circle back on how to handle this later.